app/controllers/applicants/omniauth_callbacks_controller.rb

```diff
@@ -17,11 +17,6 @@ def true_layer
       redirect_to gather_citizens_accounts_path
     end
 
-    def failure
-      set_flash_message(:error, :failure, kind: 'TrueLayer', reason: 'Process cancelled')
-      redirect_to citizens_consent_path
```

Comment:
why we don't need this method anymore?

Reply:
With devise handing the omniauth failures, the failures are redirected back to the omniauth callbacks controller's `failure` action. 

When omniauth was stripped out of the Devise config failures were being sent back to `auth/failure` and I added a separate controller to handle that.